lib/srv/reexec.go

```diff
@@ -138,6 +165,24 @@ func RunCommand() (io.Writer, int, error) {
 			return errorWriter, teleport.RemoteCommandFailure, trace.BadParameter("pty and tty not found")
 		}
 		errorWriter = tty
+		ttyName, err = os.Readlink(tty.Name())
+		if err != nil {
+			return errorWriter, teleport.RemoteCommandFailure, trace.BadParameter("Failed to resolve tty soft link: %v", err)
+		}
+		soft, err := createUaccSession(ttyName, &c)
+		if err != nil {
+			if soft {
+				log.WithError(err).Warnf("Failed to register closed interactive session for tty %s in the system account database with error %s", ttyName, trace.Wrap(err).Error())
```

Comment:
nit: if you wrap the error into the log message, there is not need to also duplicate it in the message itself. Here and below.